challenges-46-50.yml

```
@@ -0,0 +1,219 @@
+    apps such as chatrooms and the likes.
+    
+    Oh, and the Planet is called `Mercury`! :p
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          Mercure is the first planet of our Solar System!
+        b: |
+          Mercure is a service that runs in a host that allows messages to be
+          pushed to web clients without having to perform an http request!
+        c: |
+          Mercure is a browser extension for Firefox and Chrome that allows
+          us to send push notifications to them!
+        d: | 
+          Both A and B are correct!
+      correct: b
+
+listen_publish:
+  question: |
+    My chat application is finally ready to become interactive! I've just
+    installed the `symfony/ux-turbo-mercure` package and set up Mercure in
+    my server and I'm ready to receive messages from other people!
+    
+    My current template is looking like this:
+    
+    ```twig
+    <turbo-stream action="update" target="chat-messages">
+        <template>
+            {{ include('chat/_messages.html.twig') }}
+        </template>
+    </turbo-stream>
+
+    <turbo-stream action="replace" target="chat-user-list">
+        <template>
+            {{ include('chat/_user-list.html.twig') }}
+        </template>
+    </turbo-stream>
+    ```
+    
+    And I now need to listen to a `new-messages` topic in Mercure in order to
+    receive them. How is the way to do it?
+  explanation: |
+    That twig function is all you need to make your page listen to that
+    Mercure connection. It's not a *visual* component (e.g. It will not be 
+    visible in any way in our page) but a functional one!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```twig
+          <turbo-stream data-mercure-topic="new-messages" action="update" target="chat-messages>
+              <template>
+                  {{ include('chat/_messages.html.twig') }}
+              </template>
+          </turbo-stream>
+          ```
+        b: |
+          ```twig
+          <turbo-frame data-mercure-topic="new-messages">
+              {{ include('chat/_messages.html.twig') }}
+          <turbo-frame>
+          ```
+        c: |
+          Both A and D are correct.
+        d: |
+          ```twig
+          <div {{ turbo_stream_listen('new-messages') }}></div>
+          ```
+      correct: d
+
+mercure_jwt:
+  question: |
+    Now that mercure is installed, ready to go and my page enables it, I have 
+    the following template in my chat page that can receive new messages from
+    other users!
+    
+    ```twig
+    <div {{ turbo_stream_listen('new-messages') }}></div>
+    <turbo-frame id="chat-messages">
+        {{ include('chat/_messages.html.twig') }}
+    </div>
+
+    <turbo-frame id="user-list">
+        {{ include('chat/_user-list.html.twig') }}
+    </div>
+    ```
+    
+    Now I need to send messages to that template in order to update new messages
```

```suggestion
    Now I need to send updates to that template in order to update new messages
```

I think you're referring to needing to send "Mercure updates" to the template/page, right? If so, that's why I changed the word.